src/test/java/com/codejoust/main/util/MockHelper.java

```diff
@@ -49,6 +49,18 @@
         return UtilityTestMethods.toObject(jsonResponse, c);
     }
 
+    public static <T> T postRequestNoToken(MockMvc mockMvc, String url, Object body, Class<T> c, HttpStatus status) throws Exception {
+        MvcResult result = mockMvc.perform(post(url)
+                .header(HttpHeaders.AUTHORIZATION, "")
```

Comment:
if making these requeset headers optional, this line could be removed (and in RoomTestMethods)

Reply:
Done ✅